project/src/main/kotlin/org/ionproject/core/home/JsonHomeController.kt

```diff
@@ -19,15 +20,21 @@ class JsonHomeController {
 
     @GetMapping("/", produces = [Media.JSON_HOME])
     fun getRoot(): JsonHome = JsonHomeBuilder(apiName)
-        .link("describedBy", specUri)
-        // course resource
-        .newResource("courses")
-        .hrefTemplate(UriTemplate("${Uri.forCourses()}${Uri.rfcPagingQuery}"))
-        .hrefVar("limit", URI("/api-docs/params/limit"))
-        .hrefVar("page", URI("/api-docs/params/page"))
-        .docs(coursesSpecUri)
-        .formats(Media.MEDIA_SIREN)
-        .allow(HttpMethod.GET)
-        .toResourceObject()
-        .toJsonHome()
+            .link("describedBy", specUri)
+            // course resource
+            .newResource("courses")
+            .hrefTemplate(UriTemplate("${Uri.forCourses()}${Uri.rfcPagingQuery}"))
+            .hrefVar("limit", URI("/api-docs/params/limit"))
+            .hrefVar("page", URI("/api-docs/params/page"))
+            .docs(coursesSpecUri)
+            .formats(Media.MEDIA_SIREN).allow(HttpMethod.GET)
+            .toResourceObject()
+            .newResource("calendar-terms")
```

Comment:
Appreciate the change, but this should've been added through a different branch.